backend/apps/github/models/user.py

```diff
@@ -1,46 +1,108 @@
-"""Github app user model."""
+"""GitHub user model and related functionality."""
 
+from django.core.exceptions import ValidationError
 from django.db import models
 
 from apps.common.models import TimestampedModel
+from apps.github.constants import (
+    GITHUB_LOGIN_REQUIRED,
+    GITHUB_NODE_ID_NOT_FOUND,
+    GITHUB_NODE_ID_REQUIRED,
+    GITHUB_USER_DATA_REQUIRED,
+)
 from apps.github.models.common import GenericUserModel, NodeModel
 
 
 class User(NodeModel, GenericUserModel, TimestampedModel):
-    """User model."""
+    """Model representing a GitHub user or organization."""
 
     class Meta:
         db_table = "github_users"
         verbose_name_plural = "Users"
+        ordering = ["login"]
+        indexes = [
+            models.Index(fields=["login"]),
+            models.Index(fields=["type"]),
+        ]
 
-    bio = models.TextField(verbose_name="Bio", max_length=1000, default="")
-    is_hireable = models.BooleanField(verbose_name="Is hireable", default=False)
-    twitter_username = models.CharField(verbose_name="Twitter username", max_length=50, default="")
+    bio = models.TextField(
+        verbose_name="Bio",
+        max_length=1000,
+        blank=True,
+        default="",
+        help_text="User's GitHub biography",
+    )
+    type = models.CharField(
+        verbose_name="User type",
+        max_length=50,
+        default="User",
+        choices=[
+            ("User", "User"),
+            ("Organization", "Organization"),
+        ],
+        help_text="Distinguishes between User and Organization accounts",
+    )
+    public_repositories_count = models.PositiveIntegerField(
```

Comment:
It already exists -- https://github.com/OWASP/Nest/blob/4082718b8bf470557c38313595295724465254bc/backend/apps/github/models/common.py#L26